app/helpers/emoji_helper.rb

```diff
@@ -0,0 +1,12 @@
+module EmojiHelper
+  def emojify(content)
+    h(content).to_str.gsub(/:([\w+\s-]+):/) do |match|
+      name = $1.tr(" ", "_").downcase
+      if emoji = Emoji.find_by_alias(name)
+        %(<img alt="#{name}" src="#{image_path("emoji/#{emoji.image_filename}")}" style="vertical-align:middle" width="20" height="20" />)
```

Comment:
A css class is prefered. Remove the inline style. Also, width and height should be set in `em` to scale with font-size.